pykern/pkcollections.py

```diff
@@ -76,7 +67,13 @@ def __setattr__(self, name, value):
         super(PKDict, self).__setitem__(name, value)
 
     def copy(self):
-        return self.__copy__()
+        """Necessary because dict.copy will return a dict rather than PKDict.
```

Comment:
The first line is always [a summary](https://peps.python.org/pep-0257/#multi-line-docstrings) then you would give a more elaborate description. Something like:

```py
"""Override `dict.copy` to ensure the class of the return object is correct.

Necessary because dict.copy will return a dict rather than PKDict. Calls `self.__class__(self)` which makes a shallow copy. Subclasses do not accept this constructor form will need to override this method.

Returns:
     object: shallow copy of self
"""
```

We normally document `object` as the return value when it can be anything.

Reply:
6089a9e08ba0a1b1ef9f12010460150ed5064269